src/program_state.inl

```diff
@@ -2,6 +2,7 @@
 
 #include "../include/hip/hcc_detail/code_object_bundle.hpp"
 #include "../include/hip/hcc_detail/hsa_helpers.hpp"
+#include "hip/hip_runtime.h"
```

Comment:
What part of the change needs this include?

Reply:
I tried this patch, I am getting this error without this include:
```
In file included from /home/aaron/git/hip/src/program_state.cpp:3:
/home/aaron/git/hip/src/program_state.inl:653:37: error: no member named 'round_up_to_next_multiple_nonnegative' in namespace 'hip_impl'
            auto offset = hip_impl::round_up_to_next_multiple_nonnegative(current_offset,align);
```